plugins/accuracy.cc

```diff
@@ -86,6 +86,11 @@ AccuracyConfigDescription()
               "test set is very large and aggregate statistics for each unique score is "
               "sufficient, for instance to generate a ROC curve. This has no effect "
               "for other values of `mode`.", false);
+    addField("accuracyOverN", &AccuracyConfig::accuracyOverN,
```

Comment:
I would make this a vector, so that you can ask for accuracyOverN: [ 1, 3, 10 ] and get all of the values that you're after.  We want accuracy to be one-pass, and in my experience of using this you always want more than one n.